DOC-909 Interpolation support removed for fields in javascript processor (#143)
redpanda-data/rp-connect-docs: modules/components/pages/processors/javascript.adoc

== Runtime

> In order to optimize code execution, JavaScript runtimes are created on demand (in order to support parallel execution) and are reused across invocations. Therefore, it is important to understand that the global state created by your programs will outlive individual invocations. In order for your programs to avoid failing after the first invocation, ensure that you do not define variables at the global scope.

Reviewer: ```suggestion
To optimize code execution, JavaScript runtimes are created on demand (in order to support parallel execution) and are reused across invocations. Therefore, it's important to understand that the global state created by your programs will outlive individual invocations. For your programs to avoid failing after the first invocation, ensure that you do not define variables at the global scope.
```

Author: Updated.
